libs/martech/attributes.js

```diff
@@ -95,7 +95,15 @@ export async function decorateSectionAnalytics(section, idx, config) {
       block.setAttribute('daa-lh', `${lhAtt}${mepMartech}`);
     } else {
       const blockName = block.classList[0] || '';
-      block.setAttribute('daa-lh', `b${blockIdx + 1}|${blockName.slice(0, 15)}${mepMartech}`);
+      let closest = block;
+      let nestedLH = '';
+      while (closest) {
```

Comment:
this looks like a forever loop. If `closest` resolves once it will always resolve, and the loop will never end, right? Is this intended?

Reply:
line 101 is `closest = closest.parentNode?.closest('[data-nested-lh]');`, so it will always run the loop once, but it immediately redefines it.  Adding an alternative below.  Tell me what you think.

Reply:
@denlight, on line 101, for each pass of the while loop, the variable closest gets reassigned to closest.parentNode?.closest('[data-nested-lh]') so closest should eventually get set to null.